website/server/controllers/api-v3/groups.js

```diff
@@ -718,4 +715,19 @@ api.inviteToGroup = {
   },
 };
 
+// Transitioning invitation storage to an array of objects (see issue #7792 for reference)
+function clearPartyInvitation(user, groupID) {
+  user.invitations.parties.forEach(function(party, index, partiesArr) {
```

Comment:
I read through the removeFromArray utility and didn't see a markModified call in it.  If it's not being handled elsewhere let's keep this bit as-is.